spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/GRU.scala

```diff
@@ -98,10 +112,12 @@ class GRU[T : ClassTag] (
       .add(ParallelTable()
         .add(Sequential()
           .add(Dropout(p))
-          .add(Linear(inputSize, outputSize)))
+          .add(Linear(inputSize, outputSize, wRegularizer = wRegularizer,
+            bRegularizer = bRegularizer)))
         .add(Sequential()
           .add(Dropout(p))
-          .add(Linear(outputSize, outputSize, withBias = false))))
+          .add(Linear(outputSize, outputSize, withBias = false,
```

Comment:
Fixed regularization Configuration for LSTM, wRegularizer for input weight, uRegularizer for hidden weight, bRegularizer for bias.